data/api_v1.go

```diff
@@ -403,6 +406,13 @@ func writeFromIterV1(ctx context.Context, p *writeFromIter) error {
 				payload["history"] = p.parametersHistory["history"]
 				datum["payload"] = payload
 			}
+			// Add the basal security profile to the pump settings
+			if datumType == "pumpSettings" && p.basalSecurityProfile != nil {
```

Comment:
Same here we may have only one test 
```suggestion
			if datumType == "pumpSettings" && (p.basalSecurityProfile != nil || p.parametersHistory != nil) {
```